### PR TITLE
feat(mobile): e2e import all possible accounts in watch only

### PR DESCRIPTION
**Pull request by ivanderkova**
## Description
I wrote E2E test for adding all possible coins and BTC account types. I also fixed some incorrect test ids and added some utility functions.

## Related Issue

Resolve

### CODE REVIEW

**Comment by Nodonisko** [comment]
cc @ondracja

File: suite-native/app/e2e/utils.ts [hunk 1]
```diff
@@ -41,17 +42,49 @@ const openExpoDevClientApp = async () => {
...
+            newInstance,
             launchArgs: APP_LAUNCH_ARGS,
         });
     }
 };
+
+export const restartApp = async () => {
+    const { configurationName } = await resolveConfig();
+
+    const isDebugTestBuild = configurationName.split('.')[2] === 'debug';
```
Comment (PeKne) [inline suite-native/app/e2e/utils.ts]: nit: what about revoming tese two lines from both `restartApp` and `openApp` functions and put it at the beginning of this file as a separate function `const isDebugTestBuild = () => {}`? We would not have to parse the string at every function separately.

**Review by PeKne — COMMENTED** [review]
So far I've checked only the E2E test source files, but it look very good. Thank you for your contribution 👏.

File: suite-native/app/e2e/utils.ts (hunk 1 above)
Comment (ivanderkova) [reply suite-native/app/e2e/utils.ts]: fixed

**Review by PeKne — APPROVED** [review]

**Pull request merged**